auth/src/main/java/com/redhat/lightblue/rest/auth/ldap/LDAPMutipleUserFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.redhat.lightblue.rest.auth.ldap;
+
+/**
+ * This simple exception is used when an unexpected behavior is found from the LDAP server where more than one entry is returned
+ *
+ * Created by lcestari on 2/23/15.
+ */
+public class LDAPMutipleUserFoundException extends Exception {
+    public LDAPMutipleUserFoundException(String message) {
```

Comment:
My bad. I asked the IDEA to generate the Exception and I didn't release it didn't used the super constructor. Fixing